tests/integration/components/frost-bunsen-form/renderers/datetime-test.js

```diff
@@ -150,84 +154,6 @@ describe('Integration: Component / frost-bunsen-form / renderer / datetime', fun
     })
   })
 
-  describe('when form explicitly enabled', function () {
```

Comment:
Disabled property is not currently supported in frost-date-time picker, can be added back later when this becomes available, will add a sudo disabled to the template as discussed.